tst/utils/NoOPAudioVideoFacade.tsx

```diff
@@ -0,0 +1,16 @@
+import { AudioVideoObserver, DefaultAudioVideoFacade } from "amazon-chime-sdk-js";
+
+export default class NoOPAudioVideoFacade extends DefaultAudioVideoFacade {
```

Comment:
Not sure we want a common one as the need might be different from hooks to hooks. How about we keep it in the same file for now and move it out if there are more tests need this object.

Reply:
I prefer keeping it in the same file. However 'jest.mock' does not work in that case. So I have to move it out.

Reply:
Figure out that we don't need to mock the class, we can just use `jest.spyOn` for `addObserver` and `removeObserver` method.

Deleted this file move `NoOPAudioVideoFacade` back to test file.